### PR TITLE
Added a feature Show Weather

### PR DESCRIPTION
**Pull request by Yashdew**
kindly refer howtorun.md and kindly generate the API from https://openweathermap.org/current .

- [x] I have read the Code Of Conduct.
        
 - [X] I have followed all the steps of submission properly.

### CODE REVIEW

**Comment by Yashdew** [comment]
@yashpaneliya can you check my PR?

**Comment by vatsal30** [comment]
Thanks,@Yashdew, for opening the pull request! 👏

I'll be reviewing your pull request thoroughly.

Star ⭐ this project and [tweet](https://twitter.com/share?text=Contributed+to+awesome+projects+at+@club_gamma&url=https://clubgamma.github.io/hacktoberfest) about your contributions and also you can join the [discord channel of club gamma](https://discord.gg/kjnp6wU)

**Comment by jayskhatri** [comment]
Hey @Yashdew please go through once [CONTRIBUTING.md](https://github.com/clubgamma/Weather-Forecaster/blob/master/CONTRIBUTING.md) and do necessary changes

Happy Hacking!🚀

**Comment by Yashdew** [comment]
kindly refer howtorun.md and kindly generate the API from https://openweathermap.org/current .

 I have read the Code Of Conduct.

 I have followed all the steps of submission properly.

**Comment by Yashdew** [comment]
kindly refer howtorun.md and kindly generate the API from https://openweathermap.org/current .

- [X] I have read the Code Of Conduct.
        
 - [X] I have followed all the steps of submission properly.

File: Yash Dewangan/templates/index.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bhai Bhai</title>
```
Comment (vatsal30) [inline Yash Dewangan/templates/index.html:6]: Please change title name and give meaningful name.

File: Yash Dewangan/templates/index.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    <div class="container has-text-centered">
+        <div class="column is-6 is-offset-3">
+            <h1 class="title">
+                Check Weather
+            </h1>
+            <h2 class="subtitle">
+                
+            </h2>
+            <div class="box">
+                <div class="field is-grouped">
```
Comment (vatsal30) [inline Yash Dewangan/templates/index.html:68]: Hey @Yashdew Please make the ui of this block properly.

File: Yash Dewangan/templates/index.html [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+              
+              <div class="box" >
+                  <article class="media">
+                      <div class="media-left">
+                          <figure class="image is-50x50">
+                              <img src="http://openweathermap.org/img/w/{{ weather.icon }}.png" alt="Image" >
+                          </figure>
+                      </div>
+                      <div class="media-content">
+                          <div class="content" >
```
Comment (vatsal30) [inline Yash Dewangan/templates/index.html:99]: Nothing Should be shown here when there is no input in submit box and for the very first time when site is open it showing None can you change it and make it appear when any city is submitted?

File: Yash Dewangan/howtorun.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<h1 align="center">How to run</h1>
+
+1-> Clone the branch repo
+2-> add the API key and update the Weather.py
+3-> run the Following command in terminal
+        pip install -r requirements.txt
```
Comment (vatsal30) [inline Yash Dewangan/howtorun.md:6]: You need to change the type of requirement.txt it shown error that file is missing so change the type of file and also create virtual environment for this project and make requirement.txt of that environment only.

**Review by vatsal30 — CHANGES_REQUESTED** [review]

**Review by vatsal30 — CHANGES_REQUESTED** [review]
PR will be merged after you update the changes

**Comment by Yashdew** [comment]
@vatsal30 kindly check I have completed all the above requirements.

**Comment by Yashdew** [comment]
@yashpaneliya @vatsal30 Can you guys check?

**Review by vatsal30 — APPROVED** [review]

**Review by vatsal30 — APPROVED** [review]

**Review by vatsal30 — APPROVED** [review]

**Comment by vatsal30** [comment]
Thanks, @Yashdew  , for opening the pull request! 👏

Star ⭐ this project and [tweet](https://twitter.com/share?text=Contributed+to+awesome+projects+at+@club_gamma&url=https://clubgamma.github.io/hacktoberfest) about your contributions and tag @clubgamma and also you can join the [discord channel of club gamma](https://discord.gg/kjnp6wU)

Ready to merge your PR!!!

**Pull request merged**